src/please.go

```diff
@@ -313,6 +325,18 @@ var buildFunctions = map[string]func() bool{
 		}
 		return false // We should never return from run.Run so if we make it here something's wrong.
 	},
+	"parallel": func() bool {
+		if success, state := runBuild(opts.Run.Parallel.PositionalArgs.Targets, true, false); success {
+			run.Parallel(state.Graph, opts.Run.Parallel.PositionalArgs.Targets, opts.Run.Parallel.Args)
+		}
+		return true // We do return from run.Parallel once it's all over.
```

Comment:
Yeah sorry did some updates, ended up needing to pass arguments to commands and refactored some other bits at the same time.

Whether these things return is all a bit arbitrary. Originally my thought was that this was easier if it just died since on success `Run` doesn't ever return, so it's kind of weird to return a bool that's only really got one possible value.